build-rev.sh

```diff
@@ -19,6 +19,19 @@ GIT_COMMIT=$(git rev-parse HEAD)
 GIT_TAG=$(git describe --tags --dirty --match=v[0-9]* 2> /dev/null || git rev-parse --short HEAD;)
 GIT_UTCTIME=$(date -u '+%Y/%m/%d %H:%M:%S')
 
+# Cross-compilation occurs when you set TARGET_TRIPLE to something other than
+# HOST_TRIPLE. You'll need to ensure the cross-compiling toolchain is on your
+# path and override the rest of the variables that immediately follow as
+# necessary. For an example, see build/builder/cmd/mkrelease, which sets these
+# variables appropriately for the toolchains baked into the builder image.
+
+HOST_TRIPLE=$($(go env CC) -dumpmachine)
```

Comment:
Just hardcode the string `x86_64-pc-linux-gnu`. `go` isn't installed in the builder image.

Reply:
Done.